packages/console/package.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "@evidence-dev/console",
+	"version": "0.0.1",
+	"scripts": {
+		"dev": "evidence dev",
+		"build": "evidence build && npm run package",
```

Comment:
Why use the evidence cli here rather than `parcel` like core-components?

Reply:
The evidence CLI is there because it's still an evidence project, like how `npm create svelte` makes a sveltekit project. Though I'm not sure if `npm run package` is needed in the build script.

Reply:
We should probably nail down if plugins should be evidence projects or not when we look at the data sources epic. I want to lean towards they shouldn't need to be, but I'd like to discuss more.